Index: .idea/kotlinc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"Kotlin2JsCompilerArguments\">\n    <option name=\"sourceMapEmbedSources\" />\n    <option name=\"sourceMapPrefix\" />\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/kotlinc.xml b/.idea/kotlinc.xml
--- a/.idea/kotlinc.xml	(revision bfefd4821b8797ad8fae2da860750b64fb8dda67)
+++ b/.idea/kotlinc.xml	(date 1659343079209)
@@ -4,4 +4,7 @@
     <option name="sourceMapEmbedSources" />
     <option name="sourceMapPrefix" />
   </component>
+  <component name="KotlinJpsPluginSettings">
+    <option name="version" value="1.7.0" />
+  </component>
 </project>
\ No newline at end of file
